OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -66,7 +68,19 @@ public RadarWidget(World world, WorldRenderer worldRenderer)
 		{
 			this.world = world;
 			this.worldRenderer = worldRenderer;
+			currentPlayer = world.LocalPlayer ?? world.RenderPlayer;
```

Comment:
Can we call `WorldOnRenderPlayerChanged(world.LocalPlayer ?? world.RenderPlayer)` at the end of `Initialize` instead of duplicating the setup here? I'm also not sure whether it is valid to start doing some of this here before `Initialize`.

Reply:
Think it would be better to extract all code inside `WorldOnRenderPlayerChanged` to a new method  except the `// Set initial terrain data` part and use that Method here. Otherwise we will try to query the CellLayer inside PlayerRadarTerrain before its initialized.